charts/private-action-runner/templates/_helpers.tpl

```diff
@@ -4,3 +4,45 @@
 {{- define "chart.roleBindingName" }} "private-action-runner-{{.}}-rolebinding" {{ end }}
 {{- define "chart.serviceName" }} "private-action-runner-{{.}}-service" {{ end }}
 {{- define "chart.secretName" }} "private-action-runner-{{.}}-secrets" {{ end }}
+
+{{/*
+Defines an RBAC rule for provided apiGroup, resource type and allowed verbs
+*/}}
+{{- define "rbacRule" }}
+- apiGroups:
+  - {{ .apiGroup }}
+  resources:
+  - {{ .resource }}
+  verbs:
+{{- range $_, $verb := .verbs }}
+  - {{ $verb }}
+{{- end }}
+{{- end }}
+
+{{/*
+Defines an RBAC "get" rule for provided apiGroup and resource type
+*/}}
+{{- define "rbacGetRule" }}
+{{- include "rbacRule" (dict "apiGroup" .apiGroup "resource" .resource "verbs" (list "get"))}}
+{{- end }}
+
+{{/*
+Defines an RBAC "list" rule for provided apiGroup and resource type
+*/}}
+{{- define "rbacListRule" }}
+{{- include "rbacRule" (dict "apiGroup" .apiGroup "resource" .resource "verbs" (list "list"))}}
+{{- end }}
```

Comment:
It depends how much granularity you want, but maybe get and list can be define together. Same for patch and update